### PR TITLE
Tan11389/fix blurred Brest imagery

### PR DESCRIPTION
**Pull request by tanneryould**
There is a naval base on the coast of Brest, France so the imagery basemap is blurred there. This PR updates our samples to display a different area or use a different basemap

### CODE REVIEW

File: ArcGISRuntimeSDKQt_QMLSamples/Analysis/ViewshedCamera/ViewshedCamera.qml [hunk 1]
```diff
@@ -39,31 +39,31 @@ Rectangle {
...
             // Set an initial viewpoint
             ViewpointCenter {
                 Point {
                     id: centerPt
-                    x: -4.49492
-                    y: 48.3808
-                    z: 48.2511
+                    x: 2.82691
+                    y: 41.98500
+                    z:  124.987
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_QMLSamples/Analysis/ViewshedCamera/ViewshedCamera.qml:57]:
```suggestion
                    z: 124.987
```

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SceneLayerSelection/SceneLayerSelection.cpp [hunk 2]
```diff
@@ -52,7 +52,7 @@ void SceneLayerSelection::componentComplete()
   m_sceneView = findChild<SceneQuickView*>("sceneView");
 
   // Create a scene with the imagery basemap
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SceneLayerSelection/SceneLayerSelection.cpp:54]:
```suggestion
  // Create a scene with the topographic basemap
```

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.cpp [hunk 3]
```diff
@@ -52,7 +52,7 @@ void DistanceMeasurementAnalysis::componentComplete()
   m_sceneView = findChild<SceneQuickView*>("sceneView");
 
   // Create a Scene with the imagery basemap
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.cpp:54]:
```suggestion
  // Create a Scene with the topographic basemap
```

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp [hunk 4]
```diff
@@ -56,15 +57,15 @@ void ViewshedCamera::componentComplete()
...
         new ArcGISTiledElevationSource(
-          QUrl("https://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer"),
+          QUrl("https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"),
           this));
   m_scene->setBaseSurface(surface);
 
-  // Add a Scene Layer
-  ArcGISSceneLayer* sceneLayer = new ArcGISSceneLayer(QUrl("https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0"), this);
-  m_scene->operationalLayers()->append(sceneLayer);
+  // create and add the Girona, Spain integrated mesh layer
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp:64]:
```suggestion
  // Create and add the Girona, Spain integrated mesh layer
```

File: ArcGISRuntimeSDKQt_QMLSamples/Analysis/ViewshedCamera/ViewshedCamera.qml [hunk 5]
```diff
@@ -39,31 +39,31 @@ Rectangle {
...
                     spatialReference: Factory.SpatialReference.createWgs84()
                 }
-                targetScale: 62.01
+                targetScale: 1
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_QMLSamples/Analysis/ViewshedCamera/ViewshedCamera.qml:60]: The target scale is overwritten by the z value in the `centerPt` but removing it causes the sceneView to load with it completely zoomed out and centered on (0,0).

**Comment by tanneryould** [comment]
@ldanzinger - the metadata checker is complaining about `"title": "Viewshed (camera)"`. Should we update the title or the metadata checker?

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp [hunk 6]
```diff
@@ -98,15 +99,8 @@ void ViewshedCamera::calculateViewshed()
...
-  const double y = 48.3808;
-  const double z = 48.2511;
-  Point pt(x, y, z, SpatialReference(4326));
-  const double heading = 344.488;
-  const double pitch = 74.1212;
-  const double roll = 0;
-  Camera camera(pt, heading, pitch, roll);
-  Viewpoint initViewpoint(pt, camera);
-  m_scene->setInitialViewpoint(initViewpoint);
+  const Point initialPt(2.82691, 41.98500, 124.987, SpatialReference::wgs84());
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp:102]: This is fine to change. Just to give some backstory to this - at one point we had feedback that when we pass a bunch of doubles into a c'tor or method, it was difficult to tell what each argument actually was - people would need to then go to the API ref just to see what the code was doing. We countered this by being more verbose in defining variables with good names. In some cases it ends up being a bit over verbose (as is the case here). The key is to strike a balance

**Review by ldanzinger — APPROVED** [review]

**Comment by ldanzinger** [comment]
> Should we update the title or the metadata checker?

@tanneryould I'd suggest we just update the checker

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp (hunk 6 above)
Comment (tanneryould) [reply ArcGISRuntimeSDKQt_CppSamples/Analysis/ViewshedCamera/ViewshedCamera.cpp]: @ldanzinger That's a good point and I've reverted the style change.

**Comment by tanneryould** [comment]
Regards this issue: https://devtopia.esri.com/runtime/common-samples/issues/2576

**Pull request merged**